server/src/main/java/com/flipkart/varadhi/web/v1/HealthCheckHandler.java

```diff
@@ -43,7 +45,7 @@ public void bringOOR() {
     @Override
     public List<RouteDefinition> get() {
         return List.of(
-            RouteDefinition.get("HealthCheck", "/v1/health-check")
+            RouteDefinition.get("HealthCheck", HANDLER_NAME, "/v1/health-check")
```

Comment:
look redundant to me.

Reply:
here for Handlers, there is a single route, but for other handlers contain multiple routes.